core/src/main/java/com/exadel/etoolbox/backpack/core/services/impl/LiveCopyServiceImpl.java

```diff
@@ -64,13 +66,18 @@ private List<String> getLiveCopies(ResourceResolver resourceResolver, String pat
                 }
                 String liveCopyPath = liveCopy.getPath();
                 if (resourceResolver.getResource(liveCopyPath + syncPath) != null) {
-                    paths.add(liveCopyPath + syncPath);
+                    actualPaths.add(liveCopyPath + syncPath);
```

Comment:
(naming) Maybe you meant "validPaths" here? The word "actual" is a synonim of "real", "not imaginary", "what did take place" And the sense here is "not broken"

Reply:
I reused this name
![image](https://github.com/exadel-inc/etoolbox-backpack/assets/94895287/d683d323-c5ee-4a6a-8bed-1253a6c4f4d2)